GeantFast/src/FCCSmearer.cc

```diff
@@ -0,0 +1,75 @@
+#include "FCCSmearer.hh"
+#include "FCCPrimaryParticleInformation.hh"
+#include "G4PrimaryParticle.hh"
+#include "G4UnitsTable.hh"
+#include "G4SystemOfUnits.hh"
+#include "G4TransportationManager.hh"
+#include "G4FieldManager.hh"
+#include "G4UniformMagField.hh"
+#include <ctime>
+
+
+//....oooOO0OOooo........oooOO0OOooo........oooOO0OOooo........oooOO0OOooo......
+FCCSmearer* FCCSmearer::fFCCSmearer = 0;
+
+//....oooOO0OOooo........oooOO0OOooo........oooOO0OOooo........oooOO0OOooo......
+FCCSmearer::FCCSmearer()
+{
+   // unsigned seed = std::chrono::system_clock::now().time_since_epoch().count();
+   unsigned seed = time(NULL);
+   fRandomEngine = new CLHEP::HepJamesRandom( seed );
+   fRandomGauss = new CLHEP::RandGauss(fRandomEngine);
+}
+//....oooOO0OOooo........oooOO0OOooo........oooOO0OOooo........oooOO0OOooo......
+FCCSmearer::~FCCSmearer()
+{}
+
+//....oooOO0OOooo........oooOO0OOooo........oooOO0OOooo........oooOO0OOooo......
+FCCSmearer* FCCSmearer::Instance()
+{
+   if(!fFCCSmearer)
```

Comment:
that's racy and a one-time leak which pollutes valgrind reports. prefer this (still non-thread-safe):

```c++
static FCCSmearer smearer;
fFCCSmearer = &smearer;
return fFCCSmearer;
```